dropwizard-auth/src/main/java/io/dropwizard/auth/Demo.java

```diff
@@ -0,0 +1,34 @@
+package io.dropwizard.auth;
+
+public class Demo {
+    public static void main(String[] args) {
+        String rpository = "testRepo";
+        int wek = 4;
+        String day;
+    
+        switch (wek) {
```

Comment:
RSPEC-131: Switch without default case
<!-- lookout footnote separator -->
_Comment made by the analyzer sonar-check._